server/routes/referrals/furtherInformationPresenter.test.ts

```diff
@@ -0,0 +1,44 @@
+import serviceCategoryFactory from '../../../testutils/factories/serviceCategory'
+import FurtherInformationPresenter from './furtherInformationPresenter'
+
+describe('FurtherInformationPresenter', () => {
+  const serviceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
+
+  describe('title', () => {
```

Comment:
When doing the additional needs and requirements page, I found it a bit tedious writing a test for each of these bits of text, so I just [stuck them into a `text` object and tested that](https://github.com/ministryofjustice/hmpps-interventions-ui/pull/54/files#diff-cac9f0e685a58d20c619a10f762a4f25fa0330682460662f140d1696fbcc5246R43). I'll see what whoever reviews that PR thinks of it but might be a simpler approach.